packages/@jsii/java-runtime-test/project/src/test/java/software/amazon/jsii/testing/ComplianceTest.java

```diff
@@ -337,6 +337,7 @@ public void exceptions() {
         try {
           calc3.add(10);
         } catch (RuntimeException e) {
+          assertEquals(RuntimeException.class, e.getClass());
```

Comment:
This is _fine_, but you could also have used the fact that catch clauses are evaluated in declaration order and done something like this (the same applies to the other places):

```suggestion
        } catch (JsiiException e) {
          // We expect a RuntimeException that is NOT a JsiiException.
          throw e;
        } catch (RuntimeException e) {
```